### ISSUE TITLE
Effect of radiative properties on temperature cannot be observed

### ISSUE THREAD

**Issue by refetaliyalcin** [uDALES/u-dales#177]
**Describe the bug**
Hi. At Case 201, I change albedo and emissivity values in walltypes.inp.201 file. When I check tdump file with Panoply I can't observe any effects on temperature. I try to add iradiation  = 3  under &physics in namoptions.201. However I get Process name: [[37896,1],0]
  Exit code:    1. It  stops running. Is radiation working? If not how can I enable it?

**Comment by samoliverowens** [comment]
Hi, which version of the code are you trying to use/which branch have you checked out?

**Comment by refetaliyalcin** [comment]
Thanks for the quick reply. I use the most recent version that I download from here. I don't know the branch name. 
I use this command
./u-dales/tools/local_execute.sh experiments/201
obtain output: tdump.201.nc
inside that, thlt is not changing with walltypes.inp.201
I removed walltypes.inp.201 to see if the code use it but it gives error without this file.

**Comment by samoliverowens** [comment]
Ok, the example simulations are only working currently for version 1 - could you please download this version (https://github.com/uDALES/u-dales/releases/tag/v1.0.0) if you want to use them. Sorry about the confusion - version 2 (the latest version) is still effectively in beta mode until the corresponding journal article is published.

**Comment by refetaliyalcin** [comment]
Thanks for the link. I compiled the first version but unfortunately I observed the same insensitivity to albedo. When I change the albedo of walltypes.inp.201 and make all of the them 0.99, I cannot observe a change at temperature in tdump.201.nc

I checked udales-namoptions-overview.md but could not find a variable that enables or disables radiation. 
I only find skyLW parameter, which is skyLW        = 201.0
